maven-plugin/src/main/java/org/sonatype/nexus/maven/staging/PrepareDeployMojo.java

```diff
@@ -0,0 +1,32 @@
+package org.sonatype.nexus.maven.staging;
```

Comment:
License header is missing. Please run this command to add the header: `./headers.sh format`